internal/cli/deployment_create.go

```diff
@@ -85,6 +97,25 @@ func (c *DeploymentCreateCommand) Run(args []string) int {
 			}
 
 			releaseUrl = releaseResult.Release.Url
+
+			// NOTE(briancain): Because executeReleaseOp returns an initialized struct
+			// of release results, we need this deep check here to really ensure that a
+			// release actually happened, otherwise we'd attempt to run a status report
+			// on a nil release
+			if releaseResult != nil && releaseResult.Release != nil &&
+				releaseResult.Release.Release != nil {
```

Comment:
I believe the crash was fixed with the bump to argmapper 0.2 so double check the depth of checking we need here.

Reply:
Another note here...we talked about this in our sync. Argmapper handles nil better and doesn't panic anymore but I think it's still worth guarding against this since there was no release at all. Otherwise we will still attempt a status report and return an empty report, so this saves us some extra func calls.